doc/ref/notifications/notification/meta-dropbox_file.rst

```diff
@@ -4,7 +4,9 @@ dropbox_file
 ^^^^^^^^^^^^
 
 Handle a new file received in the 'files/dropbox' folder of a site. 
-Unhandled files are deleted after a hour. 
+Unhandled files are deleted after an hour. If the handler returns 'ok' then 
+the file is moved from the files/processing folder to files/handled. 
+folder. 
```

Comment:
This seems an unrelated change, just checking, maybe it wasn't intentional?

Reply:
Yes, it just tags along.